_posts/2021-04-14-accessing-the-relay-store.markdown

```diff
@@ -0,0 +1,260 @@
+---
+layout: epic
+title: "A Cool Way to Access the Relay Store"
```

Comment:
Maybe incorporate something about the checklist task into the title (in addition)?

Reply:
I'd suggest adding clarity to the title on what makes this way of accessing useful. "Accessing the Relay Store Without A Mutation" or something. I think this article is likely to be especially useful for people stuck on the same problem -- for scan-ability in search results (and to boost our ranking in the search results?) I think the title should describe the problem you solved.

Reply:
I like that! And agree, I want the title to convey the value prop.